cpp/include/cuvs/neighbors/brute_force.hpp

```diff
@@ -375,4 +383,230 @@ void search(raft::resources const& handle,
  * @}
  */
 
+/**
+ * @defgroup bruteforce_cpp_index_serialize Bruteforce index serialize functions
+ * @{
+ */
+/**
+ * Save the index to file.
+ *
+ * Experimental, both the API and the serialization format are subject to change.
```

Comment:
We may be able to remove this now (everywhere). This API has been around for awhile in RAFT and in the other indexes.

Reply:
I removed the mention of experimental, but I still kept the mention that the serialization format can be subject to changes to not have users expect retro-compatibility.